server/registry/actions_storage_test.go

```diff
@@ -0,0 +1,47 @@
+// Copyright 2022 Google LLC. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package registry
+
+import (
+	"context"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+	"google.golang.org/protobuf/testing/protocmp"
+	"google.golang.org/protobuf/types/known/emptypb"
+)
+
+func TestGetStorage(t *testing.T) {
+	ctx := context.Background()
+	server := defaultTestServer(t)
+
+	req := &emptypb.Empty{}
+
+	resp, err := server.GetStorage(ctx, req)
+	if err != nil {
+		t.Fatalf("GetStorage(%+v) returned error: %s", req, err)
+	}
+
+	// Ensure that we get the set of tables that we expect.
+	// Tables should be returned in alphabetical order.
```

Comment:
This isn't mentioned in the API documentation. IMO we shouldn't implement or rely on it unless we make it a feature. If it's just for reliable comparison in tests we can use [cmpopts.SortSlices](https://pkg.go.dev/github.com/google/go-cmp/cmp/cmpopts#SortSlices).

Reply:
OK, SGTM - I'll remove the sort and fix the test as you suggest.

Reply:
Backtracking on this, I realized that the table ordering could be handled by the SQL server with a trivial change on our side, and I think that consistent ordering makes the results more human-readable. So I'm suggesting that this API is for more than just testing - I think human users could also use it to diagnose problems in deployed instances such as accidentally dropped or renamed tables.

To the point about documentation, I've added a comment in the protos indicating that collection names are alphabetized. Thanks!